CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# This CODEOWNERS file denotes the project leads
+# and encodes their responsibilities for code review.
+
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# The format is described: https://github.blog/2017-07-06-introducing-code-owners/
+
+
+# These owners will be the default owners for everything in the repo.
+*       @talltree @wenjing
```

Comment:
@talltree @wenjing i put you two as the default owners. But I'm not sure who the right people are here.

Reply:
That's a good question. I would also add @darrellodonnell. We can always update this.